### ISSUE TITLE
table 使用scroll属性，控制台抛出ResizeObserver loop limit exceeded

### ISSUE THREAD

**Issue by guoqingfeng926** [ant-design/ant-design#43814]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/compassionate-pine-67hcgr?file=/src/index.js)

### Steps to reproduce

打开控制台，会有ResizeObserver loop limit exceeded报错

### What is expected?

报错不再抛出

### What is actually happening?

报错不再抛出

| Environment | Info |
| --- | --- |
| antd | 4.22.6 |
| React | 16.130 |
| System | mac |
| Browser | 谷歌 |

**Comment by focant** [comment]
5系列版本也有这个问题，简单看了一下，rc-table里面的ResizeObserver 的问题？看起来年代久远呀

**Comment by focant** [comment]
重写可解决

**Comment by focant** [comment]
重写ResizeObserver可解决
// 修复报错ResizeObserver loop limit exceeded
class CalmResizeObserver extends ResizeObserver {
  constructor(callback) {
    super((entries, observer) => {
      requestAnimationFrame(() => {
        callback(entries, observer);
      });
    });
  }
}
window.ResizeObserver = CalmResizeObserver;

**Comment by focant** [comment]
重写ResizeObserver解决
// 修复报错ResizeObserver loop limit exceeded
class CalmResizeObserver extends ResizeObserver {
  constructor(callback) {
    super((entries, observer) => {
      requestAnimationFrame(() => {
        callback(entries, observer);
      });
    });
  }
}
window.ResizeObserver = CalmResizeObserver;

**Comment by yoyo837** [comment]
Duplicate of #26621?

**Comment by bravoy93** [comment]
I recently migrated to version 5.13.1, and on several occasions, I encounter the error `ResizeObserver loop completed with undelivered notifications.` While it doesn't appear in production, it hinders development as I have to close the error modal constantly to continue. 

Any idea about the reason or how to solve it? I emphasize that this issue started right after upgrading to version 5.13.1.

**Comment by vidhanshu** [comment]
> I recently migrated to version 5.13.1, and on several occasions, I encounter the error `ResizeObserver loop completed with undelivered notifications.` While it doesn't appear in production, it hinders development as I have to close the error modal constantly to continue.
> 
> Any idea about the reason or how to solve it? I emphasize that this issue started right after upgrading to version 5.13.1.
> 
> React: 18.2.0 Antd: 5.13.1 OS: Sonoma 14.2.1 Browsers: Opera, Safari

Yes same here, especially In table component, the log channel is flooded with this error, liek 1000+ logs for this error

**Comment by grapho** [comment]
+1 for this `ResizeObserver loop completed with undelivered notifications` on the Table component.

In my particular case I narrowed it down to using the `scroll` attribute to define a dynamic width for fixed columns and horizontal/vertical scrolling.

I'm sure there are other components in the library wich also use dynamic positioning and resizing based on resize observer pattern

**Comment by mcshaman** [comment]
I encounter this issue with a few components. [The issue seems to be with re-resize-observer.](https://github.com/react-component/resize-observer/issues/236)

**Comment by zombieJ** [comment]
Tested in lastest version and seems fixed in v6 already: https://codesandbox.io/p/sandbox/sweet-shannon-67np6l?file=%2Fsrc%2Findex.js

@mcshaman v4 use `rc-resize-observer` and you can try `@rc-component/resize-observer`. Pls feel free to create new issue if needed.

this issue will be closed : )

**Comment by mcshaman** [comment]
@zombieJ The issue I linked uses the latest version of `@rc-component/resize-observer` in the codesandbox demo
